kuma/core/utils.py

```diff
@@ -284,3 +286,9 @@ def chord_flow(pre_task, tasks, post_task):
         return chain(*tasks)
     else:
         return chain(pre_task, chord(header=tasks, body=post_task))
+
+
+def limit_banned_ip_to_0(group, request):
+    if IPBan.objects.active(ip=get_ip(request)).count() > 0:
+        return "0/s"
+    return "60/m"
```

Comment:
This *definitely* needs caching, no way around it. This is going to be called *a lot* and no matter how optimized the MySQL cluster is, counts and checks are slow. Merge #3115 and add a django-cacheback job to handle the cache invalidation nicely.